backend/src/test/java/com/htecgroup/skynest/service/impl/UserServiceImplTest.java

```diff
@@ -73,6 +75,23 @@ void setUp() {
             roleWorkerEntity,
             null);
 
+    otherWorkerEntity =
+        new UserEntity(
+            UUID.randomUUID(),
```

Comment:
I think you can use enabledWorkerEntity, no need to create another one.

Reply:
I used it when I needed to check that other user entity is returned when admin calls get user. I cant check the constraint with the same entity. I'll take a look anyway.